docs/api/browser-view.md

```diff
@@ -46,29 +46,29 @@ Objects created with `new BrowserView` have the following properties:
 
 A [`webContents`](web-contents.md) object owned by this view.
```

Comment:
I think this one was fixed in https://github.com/electron/electron/pull/9427 and should probably be backported as well.